packages/webpack-cli/lib/groups/StatsGroup.js

```diff
@@ -5,7 +5,7 @@ const { logger } = require('@webpack-cli/logger');
  */
 class StatsGroup extends GroupHelper {
     static validOptions() {
-        return ['minimal', 'none', 'normal', 'verbose', 'errors-warnings', 'errors-only'];
+        return ['minimal', 'none', 'normal', 'verbose', 'errors-warnings', 'errors-only', 'detailed'];
```

Comment:
@rishabh3112 All values are valid https://github.com/webpack/webpack/blob/master/schemas/WebpackOptions.json#L2605

Reply:
Why not extract it from there ?

Reply:
@rishabh3112 We will do it late, it is part of task - `Args from webpack`

Reply:
Okay!